st2common/st2common/util/jinja.py

```diff
@@ -180,3 +187,17 @@ def is_jinja_expression(value):
             return True
 
     return False
+
+
+def convert_jinja_to_raw_block(value):
+    if isinstance(value, dict):
+        return {k: convert_jinja_to_raw_block(v) for k, v in six.iteritems(value)}
+
+    if isinstance(value, list):
+        return [convert_jinja_to_raw_block(v) for v in value]
+
+    if isinstance(value, six.string_types):
+        if JINJA_REGEX_PTRN.findall(value) or JINJA_BLOCK_REGEX_PTRN.findall(value):
```

Comment:
For example, if variable contains something like that: "foo bar {{ some_jinja_var }} bar foo" whole thing will be wrapped into "raw".

Just trying to understand if this is desired and if it matters if we wrap the whole string or just the part (or multiple parts) where Jinja expression is located?